contrib/face_recognition.lua

```diff
@@ -422,14 +433,14 @@ local widgets = {
 if dt.configuration.running_os == "windows" or dt.configuration.running_os == "macos" then
   table.insert(widgets, df.executable_path_widget({"face_recognition"}))
 end
-table.insert(widgets, dt.new_widget("section_label"){ label = _("processing options")})
-table.insert(widgets, fc.tolerance)
-table.insert(widgets, fc.num_cores)
-table.insert(widgets, fc.export_format)
-table.insert(widgets, dt.new_widget("box"){
-  orientation = "horizontal",
-  dt.new_widget("label"){ label = _("width  ")},
-  fc.width,
+  table.insert(widgets, dt.new_widget("section_label"){ label = _("processing options")})
+  table.insert(widgets, fc.tolerance)
+  table.insert(widgets, fc.num_cores)
+  table.insert(widgets, fc.export_format)
+  table.insert(widgets, dt.new_widget("box"){
+    orientation = "horizontal",
+    dt.new_widget("label"){ label = _("width  ")},
+    fc.width,
```

Comment:
Did you mean to indent the above lines?  They shouldn't be.

Reply:
I used an online beautifier to make sure all my changes were correctly indented. It also changed some of your code below “-- build the interface”. I will rollback it.